components/layout/app-footer.tsx

```diff
@@ -0,0 +1,81 @@
+import { GitHubLogoIcon, TwitterLogoIcon } from '@radix-ui/react-icons';
+import { Flex, Link, Text } from '@radix-ui/themes';
+import React from 'react';
+
+import { AppContent } from './app-content';
+
+const FOOTER_COMPANY_LINKS = [
+  {
+    href: 'https://experiment.st',
```

Comment:
https://linear.app/experiment-station/issue/SUPA-27/add-privacy-policy-and-terms-of-service